modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavaClientCodegen.java

```diff
@@ -746,18 +790,38 @@ public void postProcessModelProperty(CodegenModel model, CodegenProperty propert
                 model.imports.add("JsonWriter");
                 model.imports.add("IOException");
             }
+            if (additionalProperties.containsKey(SERIALIZATION_LIBRARY_JSONB)) {
+                //TODO: Fill this out
```

Comment:
todo

Reply:
I left it intentionally blank as a marker for future optimization. But indeed, it should be removed or stated to be so.